### PR TITLE
Create NConsecutiveValues and TwoPairsIdentifier Class to identify tw…

### PR DESCRIPTION
**Pull request by sergioale9**
…o pairs in hand

### CODE REVIEW

**Review by J0c4 — CHANGES_REQUESTED** [review]

File: src/main/java/jalau/at18/katas/pokerhands/sergio/ThreeOfKindIdenfier.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package jalau.at18.katas.pokerhands.sergio;
+
+public class ThreeOfKindIdenfier {
```
Comment (J0c4) [inline src/main/java/jalau/at18/katas/pokerhands/sergio/ThreeOfKindIdenfier.java:3]: typo `Identifier`

File: src/test/java/jalau/at18/katas/pokerhands/sergio/FullHouseIdentifierTest.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+import org.junit.Test;
+
+public class FullHouseIdentifierTest {
+    @Test
+    public void shouldIdentifyTFullHouse() {
+        FullHouseIdentifier identifier = new FullHouseIdentifier();
+
+        PokerHand fullHouseHand = new PokerHand(
+                new Card[] {new Card(CardValue.FOUR, 'C'), new Card(CardValue.FOUR, 'D'), new Card(CardValue.FOUR, 'S'),
```
Comment (J0c4) [inline src/test/java/jalau/at18/katas/pokerhands/sergio/FullHouseIdentifierTest.java:14]: Still need to fix indentation

**Review by J0c4 — CHANGES_REQUESTED** [review]

**Comment by J0c4** [comment]
Hello @sergioale9 , any updates on my previous comments?

File: src/test/java/jalau/at18/katas/pokerhands/sergio/StraightFlushIdentifierTest.java [hunk 3]
```diff
@@ -7,22 +7,23 @@
...
-        StraightFlushIdentifier identifier = new StraightFlushIdentifier();
+        @Test
+        public void shouldIdentifyStraightFlushHand() {
+                StraightFlushIdentifier identifier = new StraightFlushIdentifier();
 
-        PokerHand straightFlushHand = new PokerHand(
-                new Card[] {new Card(CardValue.TWO, 'C'), new Card(CardValue.THREE, 'C'), new Card(CardValue.FOUR, 'C'),
-                        new Card(CardValue.FIVE, 'C'), new Card(CardValue.SIX, 'C'), });
-        assertTrue(identifier.identify(straightFlushHand));
+                PokerHand straightFlushHand = new PokerHand(new Card[] {new Card(CardValue.TWO, 'C'),
```
[the remark this answers is not in the document]
Comment (sergioale9) [reply src/test/java/jalau/at18/katas/pokerhands/sergio/StraightFlushIdentifierTest.java]: Jose I solved the problem with indentation in test files

File: src/main/java/jalau/at18/katas/pokerhands/sergio/NConsecutiveValues.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
+package jalau.at18.katas.pokerhands.sergio;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class NConsecutiveValues extends CardsPattern {
```
[the remark this answers is not in the document]
Comment (sergioale9) [reply src/main/java/jalau/at18/katas/pokerhands/sergio/NConsecutiveValues.java]: I changed the class name to SameValuePattern i think thats a good name

File: src/main/java/jalau/at18/katas/pokerhands/sergio/ThreeOfKindIdenfier.java (hunk 1 above)
Comment (sergioale9) [reply src/main/java/jalau/at18/katas/pokerhands/sergio/ThreeOfKindIdenfier.java]: i fixed the name of this file

File: src/test/java/jalau/at18/katas/pokerhands/sergio/FullHouseIdentifierTest.java (hunk 2 above)
Comment (sergioale9) [reply src/test/java/jalau/at18/katas/pokerhands/sergio/FullHouseIdentifierTest.java]: fixed

**Comment by sergioale9** [comment]
Jose, I fixed the issues in my code and files.

**Review by J0c4 — APPROVED** [review]

**Pull request merged**